src/covid_model_seiir_pipeline/pipeline/diagnostics/model/plot_version.py

```diff
@@ -74,20 +75,34 @@ def build_cache(self, cache_dir: Path, cache_draws: List[str]):
         (self._cache / 'miscellaneous').mkdir()
 
         for measure in [*MEASURES.values(), *COVARIATES.values()]:
-            summary_data = self.pdi.load_output_summaries(self.scenario, measure.label)
-            summary_data.to_parquet(self._cache / 'summaries' / f'{measure.label}.parquet', compression=None)
+            try:
+                summary_data = self.pdi.load_output_summaries(self.scenario, measure.label)
+                summary_data.to_parquet(self._cache / 'summaries' / f'{measure.label}.parquet', compression=None)
```

Comment:
suggestion try/except/else for situations like these

Reply:
That's a much better idea.